frontend/app/src/components/accounts/balances/NonFungibleBalances.vue

```diff
@@ -188,18 +190,26 @@ const {
 } = useHistoryPaginationFilter<
   NonFungibleBalance,
   NonFungibleBalancesRequestPayload,
-  NonFungibleBalance
->(null, true, useEmptyFilter, fetchNonFungibleBalances, {
-  onUpdateFilters(query) {
-    set(ignoredAssetsHandling, query.ignoredAssetsHandling || 'exclude');
-  },
-  extraParams,
-  defaultSortBy: {
-    pagination: 'name',
-    pageParams: ['name'],
-    pageParamsAsc: [true]
+  NonFungibleBalance,
+  Collection<NonFungibleBalance>
```

Comment:
In this case, do we need to specify it here explicitly? since we are using a default Generic value?

Also maybe for this case we would use a `V = T` default and at least when we have a default Collection and no transformation it would get us rid of the two extra types in the generic

Reply:
this I didn't consider, I'd refactor it now.